src/Service/ActivityPub/ApHttpClient.php

```diff
@@ -280,22 +285,37 @@ private function getActorObjectImpl(string $apProfileId): ?string
         return $response->getContent(false);
     }
 
+    /**
+     * Remove actor object from cache.
+     *
+     * @param string $apProfileId AP profile ID to remove from cache
+     */
     public function invalidateActorObjectCache(string $apProfileId): void
     {
         $this->cache->delete($this->getActorCacheKey($apProfileId));
     }
 
+    /**
+     * Remove collection object from cache.
+     *
+     * @param string $apAddress AP address to remove from cache
+     */
     public function invalidateCollectionObjectCache(string $apAddress): void
     {
-        $this->cache->delete('ap_collection'.hash('sha256', $apAddress));
+        $this->cache->delete($this->getCollectinCacheKey($apAddress));
     }
 
     /**
+     * Retrieve AP collection object. First look in cache, then try to retrieve from AP server.
+     * And finally, save the response to cache.
+     *
+     * @return Response body
```

Comment:
I don’t think this return type matches what is actually being returned

Reply:
Its not the type its a description of what it returns. But its correct. getCollectionObjectImpl returns the response body via $response->getContent. Its getting cached if there is a cache hit. But apart from that, it's just a response body.

Reply:
I see. So the problem is that the first symbol after `@return` is expected to be a data type, such as `array|null`. The description can go after that

Reply:
AH yes you're right. I fix it now for real.